megamek/src/megamek/common/weapons/ASEWMissileWeaponHandler.java

```diff
@@ -88,24 +83,22 @@ protected void handleEntityDamage(Entity entityTarget,
             r = new Report(3470);
             r.subject = subjectId;
             r.addDesc(entityTarget);
-            vPhaseReport.add(r); 
+            vPhaseReport.add(r);
         }
-        //Large craft suffer a to-hit penalty for the location struck. 
-        if (entityTarget instanceof Dropship) { 
-            Dropship d = (Dropship) entityTarget;
+        //Large craft suffer a to-hit penalty for the location struck.
+        if (entityTarget instanceof Dropship d) {
             int loc = hit.getLocation();
             d.setASEWAffected(loc, 2);
             //Report the arc affected by the attack and the duration of the effects
             r = new Report(3472);
             r.subject = subjectId;
             r.add(entityTarget.getLocationAbbr(hit));
```

Comment:
I know it's a short use one but.... could we not use one letter variable names here? If it's something like a stream, I don't see an issue as it's a one liner, maybe a loop counter variable. 

I think we still need to define out where the line is for this. Open for discussion.

Reply:
ah yes. This was an IDEA warning and one-click change. I simply didnt look at it. I'll change it but I think we should not overdo things when it's legacy issues. Sometimes you really want to update code but then it can obscure what the PR does and other times you don't want to update every single legacy issue.

Reply:
Fair, understandable, and I agree.